csfieldguide/templates/interactives/waterfall-process.html

```diff
@@ -0,0 +1,84 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <!-- NOTE: Uninteractive must be displayed in an iframe -->
+  <div class="row justify-content-center waterfall-title">
+    {% trans 'The Waterfall process' %}
+  </div>
+  <div class="container-fluid mb-3 waterfall-process">
+    <div class="row">
+      <!-- p-1 and similar are to overwrite _grid-framework.scss values -->
+      <div class="col-3 pr-1 pb-1">
+        <div id="analysis" class="waterfall-box">
+          {% trans 'Analysis' %}
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-3 pr-1"></div>
+      <div class="col-3 p-1">
+        <div id="design" class="waterfall-box">
+          {% trans 'Design' %}
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-6 pr-1"></div>
```

Comment:
Are you able to use offset classes instead of creating empty divs? 
https://getbootstrap.com/docs/4.1/layout/grid/#offsetting-columns

Your uninteractive looks really nice though and I know it's probably a pain to try restructure the html.